### DISCUSSION TITLE
Replace pyright with jedi_language_server for Python

### DISCUSSION THREAD

**Discussion by ChenZhongPu** [LazyVim/LazyVim#1114] [Ideas]
Currentlty, it uses `pyright` as the LSP for Python, however `pyright` can only provide few package information for *hover* action (`Shift-k`) in neovim.

<img width="358" alt="image" src="https://github.com/LazyVim/LazyVim/assets/2310958/45274bb9-9079-49dd-b9dd-3b66edf23ca6">

I suggest replacing it with `jedi_language_server`, which is better in neovim.

<img width="1072" alt="image" src="https://github.com/LazyVim/LazyVim/assets/2310958/73f909de-d210-41ed-a027-15219a771668">

**Comment by fredrikaverpil** [comment]
you already posted about this in https://github.com/LazyVim/LazyVim/discussions/1113 ?

**Reply by ChenZhongPu** [reply]
Yes. Due to the network issue, I didn't realize the last post was summited. I closed this one.
